### ISSUE TITLE
Component is auto-focusing after loading

### ISSUE THREAD

**Issue by lf94** [onfido/onfido-sdk-ui#1539]
I'm going to open a PR for this; this issue is to quickly record that there is a problem.

```
    onfido-sdk-ui/src/components/PageTitle/index.tsx:29.1,29.57
    containerRef.current && containerRef.current.focus()
```
The PageTitle component is causing the page to scroll to it after mounting. This is problematic as the component is loaded with other information the user needs to read and fill out before reaching the component.

**Comment by Ludovic71** [comment]
Same issue here, any change that this issue is being fixed soon ? If there is some accessibility concerns, one solution could be to pass a props to decide to auto-focus on the title or not. Thanks!

**Comment by lf94** [comment]
@Ludovic71 I don't work for Onfido but I'll probably do this today, thank you for reminding me. :slightly_smiling_face:

**Comment by lf94** [comment]
I notice that `PageTitle` is everywhere, and would involve a bunch of changes. I'm not sure if Onfido would prefer to pass a "focused" property all over, or to use some sort of global state to pass it around.

Anyone?

**Comment by Ludovic71** [comment]
Thanks @lf94 , sorry I indeed though you work for Onfido. 

For this question about Welcome Panel, we don't use it for example and directly start at the document Panel. Also does a focus on a span title really makes sense, instead of actionable element like button/input ? 

But I agree that it would be a breaking change for people relying on this focus and passing a props to decide weather need to focus or not (enabled by default to keep backward compatibility) seems the best choice ?

**Comment by kopijunkie** [comment]
Hi @lf94 

Thank you for your PR contribution to address this issue. 

We did indeed implemented the focus on PageTitle for accessibility reasons to have screen reader focus start at the PageTitle when a user is stepping through each screen on the SDK flow. 

Just to be sure that we understand the issue correctly, is this UX causing your site/app to scroll to where this element is on initialising the SDK but this is not desirable for in yours & @Ludovic71 case as there is content on your site/app that you'd like the user to see first?

**Comment by Ludovic71** [comment]
@kopijunkie , yes for our case we have TextBoxes to fill in User Name and LastName in a form. The auto focus should remain on the first TextBox or the TextBox users are typings, but because of this pageTitle.focus() there is an annoying behavior that users can't type in the textBox anymore until re-select it. 

FYI, we dont use the welcome Panel and  1st panel to show is current document but can be configurable or different based on some user  data. So it's important that we decide to not have onfido step-wizard auto-focus when it's loaded

**Comment by lf94** [comment]
> is this UX causing your site/app to scroll to where this element is on initialising the SDK but this is not desirable for in yours & @Ludovic71 case as there is content on your site/app that you'd like the user to see first?

Yes.
